src/components/Project/SpendingStats.tsx

```diff
@@ -84,16 +84,18 @@ export default function SpendingStats({
         </div>
 
         <div>
-          <Tooltip
-            title={<Trans>Balance of the project owner's wallet.</Trans>}
-          >
-            <Space>
-              <Balance address={ownerAddress} />
-              <Trans>
-                <CrownFilled /> owner balance
-              </Trans>
-            </Space>
-          </Tooltip>
+          <Space>
+            <Balance address={ownerAddress} />
+            <Tooltip
```

Comment:
unrelated fix. makes tooltip only apply to the `owner balance` text (`<Balance />` already has its own tooltip)